website/docs/installation/ferretdb/deb.md

```diff
@@ -27,6 +27,10 @@ For more information about its configuration, please take a look at [systemd con
 FerretDB does not automatically install PostgreSQL and DocumentDB extension,
 see DocumentDB extension DEB package [installation guide](../documentdb/deb.md).
 
+:::tip
+The latest FerretDB package works best with [DocumentDB v0.102.0-ferretdb-2.0.0-rc.2](https://github.com/FerretDB/documentdb/releases/tag/v0.102.0-ferretdb-2.0.0-rc.2).
+:::
```

Comment:
Do we plan to update this page every release? This sounds like a lot of chore. It's easy to forget without having it part of checklist of release.

Is there a way to recommend, without having to change this text every release?

Reply:
Thanks @chilagrow It's probably better without the full release tag.

Reply:
It might not be the latest: we might have a new DocumentDB release while we still work on a new FerretDB release.

Instead, the FerretDB release page on GitHub will say what DocumentDB release it should use. Like now: https://github.com/FerretDB/FerretDB/releases/tag/v2.0.0-rc.2